feat(content): minor improvements (#2530)
segmentio/segment-docs: src/connections/destinations/catalog/marketo-static-lists/index.md

![](images/personas-add-emailtrait.png)

> A user with multiple email addresses will appear once per email address in the Marketo lists.

Reviewer: @segmentJason what does this sentence mean? Is this saying that a user that has multiple email addresses has each email appear once in the Marketo lists?

Author: @stayseesong Yes, that is what it points out. Maybe an unnecessary change. "addresses appears" seemed awkward to read 😄

Reviewer: ```suggestion
If a user has multiple email addresses, each address appears once in the Marketo lists. 
```

Reviewer: @segmentJason how's this☝️

Author: Works for me! Thanks @stayseesong
